packages/app/src/app/pages/Dashboard/Components/Selection/ContextMenus/SandboxMenu.tsx

```diff
@@ -112,7 +111,7 @@ export const SandboxMenu: React.FC<SandboxMenuProps> = ({
       position={position}
       style={{ width: 200 }}
     >
-      {isTemplate && activeWorkspaceAuthorization !== 'READ' ? (
+      {isTemplate && userRole !== 'READ' ? (
```

Comment:
should we abstract this behind a isViewer or a more complex `allowedCapabilities` object similar with what we have in the v2 editor?